templates/SilverStripe/Forms/CompositeField.ss

```diff
@@ -1,14 +1,9 @@
-<$Tag class="form-group CompositeField {$ExtraClass.ATT} <% if $ColumnCount %>multicolumn<% end_if %>">
-    <% if $Tag == "fieldset" && $Legend %>
-        <legend>{$Legend.XML}</legend>
-    <% end_if %>
-    <% loop $FieldList %>
-        <% if $ColumnCount %>
-            <div class="column-{$ColumnCount.ATT} {$FirstLast.ATT}">
-                {$Field}
-            </div>
-        <% else %>
+<% loop $FieldList %>
+    <% if $ColumnCount %>
+        <div class="column-{$ColumnCount.ATT} {$FirstLast.ATT}">
             {$Field}
-        <% end_if %>
-    <% end_loop %>
-</$Tag>
+        </div>
+    <% else %>
+        {$Field}
+    <% end_if %>
+<% end_loop %>
```

Comment:
Sorry, premature approval.

This template is the same now as the one in framework, but I note that the template in the cms-forms theme for admin has markup that's more similar to the original. Can you please investigate which of those two is more appropriate, and if it's framework then delete this template and let it inherit through, if it's CMS forms then adjust this template to suit it?

The purpose of overriding these templates in the theme is (A) to improve accessibility (some of which has been done in SS4 core now) and (B) to style the form fields for Bootstrap 3. This template originally had some bootstrap classes, but doesn't in the proposed version so probably won't provide any value any more.